io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -213,7 +213,14 @@ public:
    ClassDefOverride(TBufferMergerFile, 0);
 };
 
+namespace Experimental {
+using TBufferMerger R__DEPRECATED(
+   6, 28, "Please use ROOT::TBufferMerger instead of ROOT::Experimental::TBufferMerger.") = ::ROOT::TBufferMerger;
+using TBufferMergerFile
+   R__DEPRECATED(6, 28, "Please use ROOT::TBufferMergerFile instead of ROOT::Experimental::TBufferMergerFile.") =
+      ::ROOT::TBufferMergerFile;
 } // namespace Experimental
+
```

Comment:
Is this really enough to ensure all current code still compiles without problems? I'd ask CMS to test first.

Reply:
how do I ask CMS to test a PR? but also, why shouldn't this be enough (they'll have deprecation warnings, but ROOT::Experimental::TBufferMerger is there)?